TestResultSummaryService/parsers/Test.js

```diff
@@ -1,7 +1,7 @@
 const Parser = require('./Parser');
 const regexRunningTest = /.*?Running test (.*?) \.\.\.\r?/;
-const regexFinishTime = /.*?Finish Time\: .* Epoch Time \(ms\)\: (.*).*/;
-const regexStartTime = /.*?Start Time\: .* Epoch Time \(ms\)\: (.*).*/;
+const regexFinishTime = /.* (.*?) Finish Time\: .* Epoch Time \(ms\)\: (.*).*/;

```

Comment:
- please use `\d+` to get the time. This is the best way to ensure that we get a number. `.*` can match anything.
- we may or may not get system time, so `.* (.*?) ` cannot be used here.